src/main/java/it/pagopa/pn/api/dto/InputSearchNotificationDto.java

```diff
@@ -0,0 +1,124 @@
+package it.pagopa.pn.api.dto;
+
+import it.pagopa.pn.api.dto.notification.status.NotificationStatus;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.time.Instant;
+
+@EqualsAndHashCode
+@ToString
+@Getter
+public class InputSearchNotificationDto {
```

Comment:
Questa classe penso che venga usata solo internamente a pn-delivery. Quindi va in pn-delivery

Reply:
Ok, viene usata anche il pn-commons. Quindi va bene qui